admin/ce/components/common/upgrade-button.tsx

```
@@ -2,14 +2,14 @@
~
~import React from "react";
~// icons
~import { SquareArrowOutUpRight } from "lucide-react";
~// ui
~import { getButtonStyling } from "@plane/ui";
~// helpers
 import { cn } from "@/helpers/common.helper";
 
 export const UpgradeButton: React.FC = () => (
-  <a></a>
+  <a />
```

Biome (1.9.4)

[error] Provide screen reader accessible content when using `a` elements.

All links on a page should have content that is accessible to screen readers.
Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies.
Follow these links for more information,
 WCAG 2.4.4
 WCAG 4.1.2

(lint/a11y/useAnchorContent)